include/nighthawk/adaptive_load/input_variable_setter.h

```diff
@@ -1,3 +1,5 @@
+#pragma once
```

Comment:
This already existed after the comment.

Reply:
Thank you for spotting these. Done in 78338dca6de640193a4ea395ed2052c832397ea8.